outputs.tf

```diff
@@ -27,3 +32,63 @@ output "ec2_instance_profile_role_name" {
   value       = "${aws_iam_role.ec2.name}"
   description = "Instance IAM role name"
 }
+
+output "description" {
+  description = "Description of the Elastic Beanstalk Environment."
+  value       = "${aws_elastic_beanstalk_environment.default.description}"
+}
+
+output "tier" {
+  description = "The environment tier specified."
+  value       = "${aws_elastic_beanstalk_environment.default.tier}"
+}
+
+output "application" {
+  description = "The Elastic Beanstalk Application specified for this environment."
+  value       = "${aws_elastic_beanstalk_environment.default.application}"
+}
+
+output "setting" {
+  description = "Settings specifically set for this Environment."
```

Comment:
Consider consistent capitalization for "Environment"

Reply:
That was copy/pasta. Good catch, though. Adjusting.